src/Message/ItemDataTrait.php

```diff
@@ -23,7 +23,7 @@ public function getItemData(ItemBag $items)
                 'type' => $item->getType(),
                 'name' => $item->getName(),
                 'quantity' => $item->getQuantity(),
-                'tax_rate' => (int) ($item->getTaxRate() * 100),
+                'tax_rate' => (int) $item->getTaxRate() * 100,
```

Comment:
This now changes the behaviour, discarding the minor units (as shown by the test case). That will likely introduce rounding and subsequent validation errors

Reply:
See also: #27 (internal reference MWW-6214)